tools/MakeRelease.py

```diff
@@ -75,14 +78,23 @@ def CreateRelease():
    try:
       zf.write(os.path.abspath(os.path.join(MODL_PATH,MODL_FILENAME)), MODL_FILENAME)
    except OSError:
-      print "MODL file does not exist"
-      releaseBuildSuccess = False
+      try:
+         zf.write(os.path.abspath(os.path.join(MODL_PATH,MODL_FILENAME_UNSIGNED)), MODL_FILENAME_UNSIGNED)
```

Comment:
Not really a requested change, but a question. What is the white line under "try:" and "if signedRelease:" for?

Reply:
I dont know what you mean by white line?